vapid.js

```diff
@@ -212,24 +264,36 @@ var vapid = {
                  */
                 return webCrypto.exportKey('raw', this._public_key)
                     .then( key => {
+                        let pubKey = this.url_btoa(key);
                         return {
                             authorization: "Bearer " + content + "." + sig,
-                            "crypto-key": "p256ecdsa=" + this.url_btoa(key),
+                            "crypto-key": "p256ecdsa=" + pubKey,
+                            publicKey: pubKey,
                         }
                     })
             })
             .catch(err => {
-                console.error("Sign error", err);
+                console.error(this.errs.enus.ERR_SIGN, err);
             })
     },
 
     signString: function(string) {
-      let alg = {name:"ECDSA", namedCurve: "P-256", hash:{name:"SHA-256"}};
-      return webCrypto.sign(alg, this._private_key, this._str_to_array(string))
-        .then(signed => {
-          let sig = this.url_btoa(signed);
-          return sig;
-        });
+        /* Sign the token for the developer Dashboard */
+        let alg = {name:"ECDSA", namedCurve: "P-256", hash:{name:"SHA-256"}};
+        let t2v = this.url_atob(string);
```

Comment:
ah, this seems to be the key I was missing in my implementation? i.e., convert the url-safe base64 string into the binary array.